compiler/rustc_resolve/src/macros.rs

```diff
@@ -22,7 +22,9 @@ use rustc_hir::def::{self, DefKind, NonMacroAttrKind};
 use rustc_hir::def_id::{CrateNum, LocalDefId};
 use rustc_middle::middle::stability;
 use rustc_middle::ty::RegisteredTools;
-use rustc_session::lint::builtin::{LEGACY_DERIVE_HELPERS, SOFT_UNSTABLE, UNUSED_MACROS};
+use rustc_session::lint::builtin::{
+    LEGACY_DERIVE_HELPERS, SOFT_UNSTABLE, UNUSED_MACROS, UNUSED_MACRO_RULES,
+};
```

Comment:
Nit: could you avoid multi-line imports?

Reply:
How should I do it? If I move it to the same line, rustfmt makes it multiline again. I could refer to `UNUSED_MACRO_RULES` via the full path, but that would be inconsistent with the other lints.

Reply:
By splitting it into multiple imports.